.github/workflows/check-dist.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
What in the world?  It looks like this repo has the same/similar renovate config to our other repos; why did it elect to use the hash here and not just update the version tag?